Cargo.lock

```diff
@@ -775,6 +775,7 @@ dependencies = [
  "ckb-hash",
  "ckb-types",
  "eaglesong",
+ "log 0.4.8",
```

Comment:
Linters Error: `Cargo.lock` is changed after build.
```suggestion
 "log 0.4.11",
```

Reply:
Roger